auxiliary.ps1

```diff
@@ -0,0 +1,104 @@
+$FILTERS = "https://go.microsoft.com/fwlink/?linkid=875139"
+$STUDIOCOMPUTERNAME = "STUDIO"
+$CLIENTCOMPUTERNAME = "CL"
+$KITTYPE = "HLK"
+$HLKKITVER = 1809

```

Comment:
Should be configurable. I am missing the place where it can be changed.

Reply:
Let's document it

Reply:
other variant is create `args.ps1` by analogy with `VirtHCK` to redefine configuration. In this case `auto_hck` will create this file with proper values

Reply:
Looks like a better option. With sample args.ps 1 in repo and documentation.

Reply:
Updated README is ok as documentation?

Reply:
Perfect